File: API/Entities/Artist.cs
namespace API.Entities
{
    public class Artist
    {
        public int Id { get; set; } 
        public string ArtistName { get; set; }
        public string ArtistDescription { get; set;}

        public int GenreId { get; set; }
        public Genre Genre { get; set; }

        public ICollection<Album> Albums { get; set; }
        public ICollection<ArtistPhoto> Photos { get; set; }
    }
}

File: client/src/app/_models/artist.ts
import {Album} from "./album";
import {Genre} from "./genre";
import {Photo} from "./photo";

export interface Artist {
  id: number;
  artistName: string;
  artistDescription: string;
  photoUrl: string;
  genreId: number;
  genre: Genre;
  albums: Album[];
  photos: Photo[];
}
